src/org/omegat/core/team2/impl/GITRemoteRepository2.java

```diff
@@ -96,9 +105,28 @@
     ProjectTeamSettings projectTeamSettings;
 
     static {
-        CredentialsProvider.setDefault(new GITCredentialsProvider());
+        installSshSessionFactory();
+        GITCredentialsProvider.install();
+    }
+
+    private static void installSshSessionFactory() {
+        SshdSessionFactory sshdSessionFactory = new SshdSessionFactoryBuilder()
+                .setPreferredAuthentications("publickey,keyboard-interactive")
+                .setHomeDirectory(FS.detect().userHome())
+                .setSshDirectory(new File(FS.detect().userHome(), ".ssh"))
```

Comment:
Does this support `_ssh` on Windows?

Reply:
No, it does not support it yet.

Reply:
Windows10 now distribute openssh as a default component in `c:\windows\system32\openssh.exe` and place ssh home directory at `%USERPROFILE%\.ssh`. It is enough to set  it `~/.ssh`.